brevent/src/main/res/values/strings.xml

```diff
@@ -178,14 +178,14 @@
     <string name="show_donation_rmb">Thanks for paying %s Brefoil</string>
     <string name="show_donation_contributor">Thanks for priceless contribution on Brevent</string>
     <string name="show_donation_play_and_rmb">Thanks for paying %1$s Brefoil via Play Store, %2$s Brefoil via non Play Store</string>
-    <string name="show_donation_play_and_contributor">Thanks for paying %s Brefoil via Play Store, and priceless contribution on Brevent</string>
-    <string name="show_donation_rmb_and_contributor">Thanks for paying %s Brefoil, and priceless contribution on Brevent</string>
+    <string name="show_donation_play_and_contributor">Thanks for paying %s Brefoil via Play Store, and priceless contribution to Brevent</string>
+    <string name="show_donation_rmb_and_contributor">Thanks for paying %s Brefoil, and priceless contribution to Brevent</string>
     <string name="show_donation_play_and_rmb_and_contributor">Thanks for paying %1$s Brefoil via Play Store, %2$s Brefoil via non Play Store, and priceless contribution on Brevent</string>
     <string name="show_donation_xposed">(Xposed)</string>
     <string name="show_donation_brefoil" />
     <string name="brevent_appops">App ops</string>
-    <string name="brevent_appops_label">Be careful, may cause apps crash</string>
-    <string name="brevent_disable">Disable Apps</string>
+    <string name="brevent_appops_label">Be careful, this may cause apps to crash</string>
```

Comment:
keep original

Reply:
"Be careful, may cause apps crash" isn't grammatically correct tho >.> but ok